pkg/config/pipeline_builder_test.go

```diff
@@ -117,6 +118,8 @@ func TestKafkaPromPipeline(t *testing.T) {
 		GroupByKeys:   api.AggregateBy{"srcAS"},
 		OperationType: "count",
 	}})
+	var expirtyTimeDuration api.Duration
```

Comment:
Typo in the word "expirty". It appears also in the other files and the title of the PR